Backend/DataAccessLayer/Database/dbContext.cs

```diff
@@ -1,28 +1,26 @@
-﻿using DataAccessLayer.Models;
-using System;
-using System.Collections.Generic;
-using System.Data.Entity;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
-using Microsoft.SqlServer.Server;
-using System.Data.SqlClient;
-
-namespace DataAccessLayer.Database
-{
-    public class DatabaseContext : DbContext
-    {
-        public DatabaseContext()
-        {
-            this.Database.Connection.ConnectionString = "Data Source=localhost;Initial Catalog=SSO;Integrated Security=True";
-        }
-
-        public DbSet<User> Users { get; set; }
-        public DbSet<Session> Sessions { get; set; }
-        public DbSet<Application> Applications { get; set; }
-        public DbSet<ApiKey> Keys { get; set; }
-        public DbSet<PasswordReset> ResetIDs { get; set; }
-
-
-    }
-}
+﻿using DataAccessLayer.Models;
+using System;
+using System.Collections.Generic;
+using System.Data.Entity;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.SqlServer.Server;
+using System.Data.SqlClient;
+
+namespace DataAccessLayer.Database
+{
+    public class DatabaseContext : DbContext
+    {
+        public DatabaseContext()
+        {
+            this.Database.Connection.ConnectionString = "Data Source=localhost;Initial Catalog=SSO;Integrated Security=True";
+        }
+
+        public DbSet<User> Users { get; set; }
+        public DbSet<Session> Sessions { get; set; }
+        public DbSet<PasswordReset> PasswordResets { get; set; }
+        public DbSet<Application> Applications { get; set; }
+        public DbSet<ApiKey> Keys { get; set; }
+    }
+}

```

Comment:
Same with this file - why is the whole file modified? Make the changes you need and leave the rest the same.